platform/mds/mds/src/main/java/org/motechproject/mds/jdo/SchemaGenerator.java

```diff
@@ -99,6 +102,8 @@ public void runMigrations(File migrationDirectory) {
         flyway.setInitOnMigrate(true);
 
         flyway.migrate();
+
+        Thread.currentThread().setContextClassLoader(cl);
```

Comment:
To create migration table. SchemaGenerator bean is created in the mds-entities.jar and flyway cannot find sql script.